### ISSUE TITLE
Is there a way to set pagination to false based on size of data ?

### ISSUE THREAD

**Issue by mahashg** [LinkedInAttic/Backbone.TableView#18]
If the data size is too small say 4 entries, is there a way to set pagination to false ? it looks ugly on UI. having pagination which has nothing associate with it.

**Comment by jpbottaro** [comment]
No, we don't have that feature. But please feel free to submit a PR for it, it sounds like a useful thing to have.
